src/components/registration/steps.jsx

```diff
@@ -948,6 +946,7 @@ class AddressStep extends React.Component {
     render () {
         let stateOptions = countryData.subdivisionOptions[this.state.countryChoice];
         stateOptions = [{}].concat(stateOptions);
+        const countryOptions = getCountryOptions(this.props.intl);
```

Comment:
this change isn't really necessary but i think it makes the code more readable and consistent